setup.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+try:
+    from setuptools import setup, find_packages
+except ImportError:
+    from distutils.core import setup
+
+
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+with open('requirements.txt') as req_file:
+    requirements = req_file.read().split('\n')
+
+with open('requirements-dev.txt') as req_file:
+    requirements_dev = req_file.read().split('\n')
+
+with open('VERSION') as fp:
+    version = fp.read().strip()
+
+setup(
+    name="vumi-http-proxy",
+    version=version,
+    description="HTTP Proxy in Python Twisted to prevent unauthorized access to blacklisted ips",
+    long_description=readme,
+    author="Carla Wilby",
+    author_email='thisiscarlawilby@gmail.co.za',
+    url='https://github.com/praekelt/vumi-http-proxy',
+    packages=[
+        find_packages(),
+    ],
+    package_dir={'': 'src'},
```

Comment:
You probably want to leave out `package_dir` because the defaults (`{'': '.'}`) match how you have things laid out now (i.e. your packages are next to your `setup.py` file and not inside a `src` folder).